src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/opentelemetry/proto/metrics/experimental/metrics_config_service.proto

```diff
@@ -0,0 +1,102 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.metrics.experimental;
+
+import "opentelemetry/proto/resource/v1/resource.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opentelemetry.proto.metrics.experimental";
+option java_outer_classname = "MetricConfigServiceProto";
+option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/metrics/experimental";
+
+// MetricConfig is a service that enables updating metric schedules, trace
+// parameters, and other configurations on the SDK without having to restart the
+// instrumented application. The collector can also serve as the configuration
+// service, acting as a bridge between third-party configuration services and
+// the SDK, piping updated configs from a third-party source to an instrumented
+// application.
+service MetricConfig {
+  rpc GetMetricConfig (MetricConfigRequest) returns (MetricConfigResponse);
+}
+
+message MetricConfigRequest{
+
+  // Required. The resource for which configuration should be returned.
+  opentelemetry.proto.resource.v1.Resource resource = 1;
+
+  // Optional. The value of MetricConfigResponse.fingerprint for the last
+  // configuration that the caller received and successfully applied.
+  bytes last_known_fingerprint = 2;
+}
+
+message MetricConfigResponse {
+
+  // Optional. The fingerprint associated with this MetricConfigResponse. Each
+  // change in configs yields a different fingerprint. The resource SHOULD copy
+  // this value to MetricConfigRequest.last_known_fingerprint for the next
+  // configuration request. If there are no changes between fingerprint and
+  // MetricConfigRequest.last_known_fingerprint, then all other fields besides
+  // fingerprint in the response are optional, or the same as the last update if
+  // present.
+  //
+  // The exact mechanics of generating the fingerprint is up to the
+  // implementation. However, a fingerprint must be deterministically determined
+  // by the configurations -- the same configuration will generate the same
+  // fingerprint on any instance of an implementation. Hence using a timestamp is
+  // unacceptable, but a deterministic hash is fine.
+  bytes fingerprint = 1;
+
+  // A Schedule is used to apply a particular scheduling configuration to
+  // a metric. If a metric name matches a schedule's patterns, then the metric
+  // adopts the configuration specified by the schedule.
+  message Schedule {
+
+    // A light-weight pattern that can match 1 or more
+    // metrics, for which this schedule will apply. The string is used to
+    // match against metric names. It should not exceed 100k characters.
+    message Pattern {
+      oneof match {
+        string equal_to = 1;       // matches the metric name exactly
```

Comment:
This was originally name `equals`, but this results in a property named `Equals` in the generated code that conflicts with the standard `Equals` method.

There may be a better way to solve this. Maybe the solution should be to rename it at the source in `opentelemetry-proto`?

This file is deemed experimental functionality, so my thought was that a manual rename is ok in the meantime. Also, none of this is currently used by our SDK.